lib/urlwatch/tests/test_handler.py

```diff
@@ -90,7 +90,7 @@ def test_pep8_conformance():
 class ConfigForTest(BaseConfig):
     def __init__(self, config, urls, cache, hooks, verbose):
         (prefix, bindir) = os.path.split(os.path.dirname(os.path.abspath(sys.argv[0])))
-        super().__init__('urlwatch', os.path.dirname(__file__), config, urls, cache, hooks, verbose)
+        super().__init__('urlwatch', os.path.dirname(__file__), os.path.dirname(__file__), config, urls, cache, hooks, verbose)
```

Comment:
Should this use a temporary folder instead of the path where test_handler is? If it's installed, it might not be writable to a user wanting to run the tests.

Reply:
Made temporary folder. Not sure why `os.path.dirname(__file__)` is used here.
note: what do you thing about renaming `urlwatch_dir` to `urlwatch_config_dir` ?